activerecord/test/schema/schema.rb

```diff
@@ -169,7 +169,7 @@
   end
 
   create_table :comments, force: true do |t|
-    t.integer :post_id, null: false
```

Comment:
How would you recommend that I check to make sure this isn't happening? 😬 Worst come to worst, I can add another model, and start fresh.